release-notes.md

```diff
@@ -4,6 +4,14 @@
 
 The following updates were made in this release:
 
+**Android Identity 1.1.1**
+
+- Identity extension now ignores any custom identifiers with null or empty ids when calling `syncIdentifier` or `syncIdentifiers` APIs as the Visitor ID Service does not support these identifiers. 
```

Comment:
We don't need to say "Identity extension" before each point as they are all under the Identity release section.

- Now ignores any custom identifiers ...
- API `syncIdentifiers` now ignores an empty Map parameter.
- Duplicate advertising identifiers are now removed from the shared state when `MobileCore.setAdvertisingIdentifier` is called with a new value.
- Now ignores the global.ssl ...

Reply:
Here are my comments/changes:

- Please add a lead-in sentence between the heading and the 1st bullet in the list. Something like "The following changes were made in this release:" to match the other sections in the release notes.

- Agree with Kevin's point about not repeating "Identity extension" for each bullet point. Reword the 1st bullet to read as follows: 
"Custom identifiers with null or empty IDs are ignored when calling `syncIdentifier` or `syncIdentifiers` APIs because the Visitor ID Service does not support these identifiers."